torch/csrc/jit/pickler.cpp

```diff
@@ -609,7 +610,7 @@ OpCode Unpickler::readInstruction() {
       stack_.emplace_back(IValue(tuple));
     } break;
     case OpCode::EMPTY_DICT:
-      stack_.emplace_back(c10::ivalue::UnorderedMap());
+      stack_.emplace_back(c10::impl::GenericDict());
```

Comment:
impl? Feels really weird here.

Reply:
It's because I don't want kernel APIs to use `GenericDict`. They should use `Dict<Key, Value>` instead. `GenericDict` is internal only for casting to/from `IValue`, and for some dict prim ops like index access that don't require to know the concrete types.